.gitignore

```diff
@@ -107,6 +107,10 @@ target/
 
 *.log
 
+# Conda packages
+
+/ci/conda/build_artifacts/
```

Comment:
`/build/conda/`?

Reply:
Oops, I think this is leftover from when I was building them directly instead of via the script/container.